src/c20_client/get_client_id.py

```diff
@@ -0,0 +1,43 @@
+"""
+File used to check for an environment variable for id and to
+get an id form server and assign the id to the enviornment
+variable if the environment variable does not yet exist
+"""
+from os import getenv
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+def request_id():
+    """
+    Ask Moravian server endpoint for id
+    Returns an id
+    """
+    response = requests.get('http://capstone.cs.moravian.edu/get_user_id')
```

Comment:
This needs error checking.  

If the call to the server fails, the program won't have a job idea, and (presumably) other calls to the server would fail anyway.

I don't see any other ways it would fail, but think through to be sure...